server/datastore/mysql/hosts.go

```diff
@@ -621,13 +658,9 @@ func (d *Datastore) AuthenticateHost(ctx context.Context, nodeKey string) (*flee
 }
 
 func (d *Datastore) MarkHostSeen(ctx context.Context, host *fleet.Host, t time.Time) error {
```

Comment:
Double checking: Seems `Datastore.MarkHostSeen` is not used at all? (`MarkHostsSeen` is the one being used).

Just a nit: we could remove the method on another PR.

Reply:
I believe it's used in tests.